apps/console/app/lib/auth/stack-auth.ts

```
@@ -52,6 +53,20 @@ const authService = {
     });
     return apiKey.value;
   },
+
+  getAccessToken() {
+    const rawCookie = getCookie("stack-access");
+    if (!rawCookie) return;
+    try {
+      const parsed = JSON.parse(decodeURIComponent(rawCookie));
+      const accessToken: unknown = Array.isArray(parsed)
+        ? parsed[1]
+        : undefined;
+      return typeof accessToken === "string" ? accessToken : undefined;
+    } catch {
+      return;
```

This looks dangerous. Why are you hiding the exception?

---

Fixed.